tests/unittests/test_commands.py

```diff
@@ -134,6 +134,28 @@ def test_text_saves_params(self):
         self.assertEqual(output_text.font, "Bitstream Vera Bold Italic")
         self.assertEqual(output_text.fontsize, 64)
 
+    @test_as_bot()
+    def test_text_bounds_property(self):
+        """
+        Check text.bounds() against expected values
+        """
+        test_fonts = {
+            "DejaVu Sans Book": (1, 87.0, 87, 14),
+            "Liberation Sans Regular": (1, 88.0, 79, 13),
+            "Bitstream Vera Sans Roman": (1, 87.0, 87, 14),
+        }
+
+        available_fonts = [f for f in fontnames() if f in test_fonts.keys()]
+        if not available_fonts:
+            raise SystemError("None of the test fonts is available in this system")
+
+        for fontname, value in test_fonts.items():
+            if not fontname in fontnames():
+                continue
+            font(fontname)
```

Comment:
This raises a question about what happens when you pass font() a fontname that doesn't exist.
Whatever happens, there should probably be a test for that at some point; fallback or raise exception etc ?
- doesn't have to block this ticket obvs.

Reply:
I think Pango will default to your system's Sans font, from what I remember. Would be nice to warn the user about this in the console

Reply:
moved this to #369